soundbay/models.py

```diff
@@ -27,6 +30,114 @@ def _choose_block_class(block):
         return getattr(importlib.import_module(module_name), class_name)
 
 
+class SqueezeNet1D(nn.Module):
```

Comment:
is there a reason to copy the model instead of importing it from torch?
And if there is a change from the torch version, could you please specify it?

Reply:
the change was the 1 input channel in the first Conv2d, but updated to a version that uses the original definition.